docs/sources/reference/components/otelcol/otelcol.processor.interval.md

```diff
@@ -0,0 +1,165 @@
+{{< /admonition >}}
+
+{{< admonition type="note" >}}
+`otelcol.processor.interval` is a wrapper over the upstream OpenTelemetry Collector `interval` processor.
+Bug reports or feature requests will be redirected to the upstream repository, if necessary.
+{{< /admonition >}}
+
+## Usage
+
+```alloy
+otelcol.processor.interval "LABEL" {
+  output {
+    metrics = [...]
+  }
+}
+```
+
+## Arguments
+
+`otelcol.processor.interval` supports the following arguments:
+
+Name          | Type       | Description                                                               | Default | Required
+------------- | ---------- | ------------------------------------------------------------------------- | ------- | --------
+`interval`    | `duration` | The interval in which the processor should export the aggregated metrics. | `"60s"` | 
+
+## Blocks
+
+The following blocks are supported inside the definition of `otelcol.processor.interval`:
+
+Hierarchy     | Block             | Description                                                                | Required
+------------- | ----------------- | -------------------------------------------------------------------------- | --------
+output        | [output][]        | Configures where to send received telemetry data.                          | yes
+debug_metrics | [debug_metrics][] | Configures the metrics that this component generates to monitor its state. | no
+
+[output]: #output-block
+[debug_metrics]: #debug_metrics-block
+
+### output block
+
+{{< docs/shared lookup="reference/components/output-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+### debug_metrics block
+
+{{< docs/shared lookup="reference/components/otelcol-debug-metrics-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+## Exported fields
+
+The following fields are exported and can be referenced by other components:
+
+Name    | Type               | Description
+--------|--------------------|-----------------------------------------------------------------
+`input` | `otelcol.Consumer` | A value that other components can use to send telemetry data to.
+
+`input` accepts `otelcol.Consumer` data for metrics.
+
+## Component health
+
+`otelcol.processor.interval` is only reported as unhealthy if given an invalid configuration.
+
+## Debug information
+
+`otelcol.processor.interval` does not expose any component-specific debug information.
+
+## Examples
+
+### Usage
+
+This example receives OTLP metrics and aggregates them for 30s before sending to the next exporter.
+
+```alloy
+otelcol.receiver.otlp "default" {
+  grpc { ... }
+  http { ... }
+
+  output {
+    metrics = [otelcol.processor.interval.default.input]
+  }
+}
+
+otelcol.processor.interval "default" {
+  output {
+    metrics = [prometheus.remotewrite.default.receiver]
+  }
+}
+
+prometheus.remote_write "default" {
+  endpoint {
+    url = env("PROMETHEUS_SERVER_URL")
+  }
+}
```

Comment:
```suggestion
otelcol.exporter.otlphttp "grafana_cloud" {
  client {
    endpoint = "https://otlp-gateway-prod-gb-south-0.grafana.net/otlp"
    auth     = otelcol.auth.basic.grafana_cloud.handler
  }
}

otelcol.auth.basic "grafana_cloud" {
  username = env("GRAFANA_CLOUD_USERNAME")
  password = env("GRAFANA_CLOUD_API_KEY")
}
```

It is not possible to send OTLP signals to `prometheus.remote_write` directly. The metrics need to go through `otelcol.exporter.prometheus` first, which will convert them from OTLP format to Prometheus format.